src/sfctl/commands.py

```diff
@@ -305,6 +305,11 @@ def load_command_table(self, args): #pylint: disable=too-many-statements
             with super_group.group('application') as group:
                 group.command('provision', 'provision_application_type')
 
+        with CommandSuperGroup(__name__, self, 'sfctl.custom_version#{}',
```

Comment:
I would split this into two different commands, one to get the target service fabric runtime version, and one to get the current version

Reply:
Would the target service fabric runtime command be useful then because right now in -h it lists the version of service fabric it is compatible with.